Loop/View Controllers/CarbEntryViewController.swift

```diff
@@ -104,8 +104,7 @@ final class CarbEntryViewController: LoopChartsTableViewController, Identifiable
                 quantity: quantity,
                 startDate: date,
                 foodType: foodType,
-                absorptionTime: absorptionTime,
-                externalID: originalCarbEntry?.externalID
```

Comment:
Isn't externalID how CarbStore knows it's an update, and not a new entry?

Reply:
No. The fact that there is an `originalCarbEntry` means that it is an update, by definition. The existence of `originalCarbEntry` eventually makes it through the workflow to `LoopDataManager.addCarbEntry` which then does the "right thing". If an `originalCarbEntry` exists, then it calls `CarbStore.replaceCarbEntry` and, if not, then `CarbStore.addCarbEntry`.